apps/mobile/src/store/storage.ts

```diff
@@ -1,19 +1,69 @@
+/* eslint-disable @typescript-eslint/no-extraneous-class */
 import { Storage } from 'redux-persist'
 import { MMKV } from 'react-native-mmkv'
+import { mapStorageTypeToIds, STORAGE_IDS, STORAGE_TYPES } from './constants'
 
-const storage = new MMKV()
+const safeStorage = new MMKV({
+  id: STORAGE_IDS.SAFE,
+})
 
 export const reduxStorage: Storage = {
   setItem: (key, value) => {
-    storage.set(key, value)
+    safeStorage.set(key, value)
     return Promise.resolve(true)
   },
   getItem: (key) => {
-    const value = storage.getString(key)
+    const value = safeStorage.getString(key)
     return Promise.resolve(value)
   },
   removeItem: (key) => {
-    storage.delete(key)
+    safeStorage.delete(key)
     return Promise.resolve()
   },
 }
+
+export class safeMMKVStorage {
+  static getLocal(key: string) {
```

Comment:
Are you expecting the key to ever be an arbitrary thing or is it going it be only `STORAGE_TYPES`?